app/helpers/budget_headings_helper.rb

```diff
@@ -12,4 +12,17 @@ def heading_link(assigned_heading = nil, budget = nil)
     link_to(assigned_heading.name, heading_path)
   end
 
+  def current_heading_map_locations(investments)
+    investments.map do |investment|
```

Comment:
This code is almost the same as the one we needed to remove at https://github.com/AyuntamientoMadrid/consul/pull/1355/files#diff-b59938c58b6939bcd1e0cf04e673f10a because it was creating huge loading problems.